python/samples/getting_started/agents/azure_ai/azure_ai_with_existing_agent.py

```diff
@@ -35,24 +24,28 @@ async def main() -> None:
         AzureCliCredential() as credential,
         AIProjectClient(endpoint=os.environ["AZURE_AI_PROJECT_ENDPOINT"], credential=credential) as client,
     ):
-        # Create an agent that will persist
-        created_agent = await client.agents.create_agent(
-            model=os.environ["AZURE_AI_MODEL_DEPLOYMENT_NAME"], name="WeatherAgent"
+        azure_ai_agent = await client.agents.create_agent(
+            model=os.environ["AZURE_AI_MODEL_DEPLOYMENT_NAME"],
+            # Create remote agent with default instructions
+            # These instructions will persist on created agent for every run.
+            instructions="End each response with [END].",
         )
 
+        chat_client = AzureAIAgentClient(project_client=client, agent_id=azure_ai_agent.id)
+
         try:
             async with ChatAgent(
-                # passing in the client is optional here, so if you take the agent_id from the portal
-                # you can use it directly without the two lines above.
-                chat_client=AzureAIAgentClient(project_client=client, agent_id=created_agent.id),
-                instructions="You are a helpful weather agent.",
-                tools=get_weather,
+                chat_client=chat_client,
+                # Instructions here are applicable only to this ChatAgent instance
```

Comment:
Would be good to clarify what is being sent to the model as a result of overriding instructions.